pallets/manta-pay/src/lib.rs

```diff
@@ -1011,7 +1024,7 @@ where
         let _ = (proof, super_key);
         for WrapPair(account_id, withdraw) in sources {
             FungibleLedger::<T>::transfer(
-                Pallet::<T>::id_from_field(encode(asset_id))
+                Pallet::<T>::id_from_field(fp_encode(asset_id).expect(FP_ENCODE))
```

Comment:
Not sure `expect` what it will affect if `expect` happens.
Or remove this `expect` with `map_err(...)?`

Reply:
+1 to error handle, otherwise add an explanation why this is impossible to hit,

Reply:
done for all method except `spend_all` and `spend_all` because the method return type is `()`.